eclipse-collections-api/src/main/java/org/eclipse/collections/api/map/MutableMap.java

```diff
@@ -184,16 +184,35 @@
     }
 
     @Override
-    <K2, V2> MutableMap<K2, V2> aggregateInPlaceBy(
-            Function<? super V, ? extends K2> groupBy,
```

Comment:
I think this might be able to be deleted as it inherits from `MutableMapIterable` which has the same implementation.